src/test/resources/keys/bad-public.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
rename these 2 files to indicate the algorithm used

Reply:
renamed to `bad-public-rsa` and `public-rsa`.